quesma/queryparser/pancake_sql_query_generation.go

```diff
@@ -105,6 +106,16 @@ func (p *pancakeSqlQueryGenerator) isPartOfGroupBy(column model.Expr, groupByCol
 func (p *pancakeSqlQueryGenerator) generateBucketSqlParts(bucketAggregation *pancakeModelBucketAggregation, groupByColumns []model.AliasedExpr, hasMoreBucketAggregations bool) (
 	addSelectColumns, addPartColumns, addGroupBys, addRankColumns []model.AliasedExpr, addRankWheres []model.Expr, addRankOrderBys []model.OrderByExpr, err error) {
 
+	if query_util.IsAnyKindOfTerms(bucketAggregation.queryType) {
+		parentCountColumn := model.WindowFunction{Name: "sum",
+			Args:        []model.Expr{model.NewFunction("count", model.NewLiteral("*"))},
```

Comment:
Do we understnad how does it count null variables, do we need extra logic?

I would also prefer to make it after group by.